python/peacock/tests/input_tab/InputTree/gold/fsp_test.i

```diff
@@ -1,27 +1,63 @@
+[Mesh]
```

Comment:
It tries to keep the same order of top level blocks, subblocks, and parameters that were in the original input file. Previously it didn't. Now the diff between the original input file and the peacock processed input file is much smaller (mostly moving comments and quotes). This allows for much easier visually verifying that the peacock version of the input file is OK.